config/config.go

```diff
@@ -559,12 +562,12 @@ func validate(c *Config) error {
 	return c.parseForwardedHeaders()
 }
 
-func (c *Config) Parse() error {
-	flag.Parse()
+func (c *Config) Parse(args []string) error {
+	c.Flags.Parse(args)
```

Comment:
Lets check error here

Reply:
Nevertheless lets return possible error here. We may also consider using `flag.ContinueOnError`